cmd/update-main_test.go

```diff
@@ -158,6 +164,59 @@ func TestGetCurrentReleaseTime(t *testing.T) {
 	}
 }
 
+// Tests user agent string.
+func TestUserAgent(t *testing.T) {
+	customUserAgent := func(mode string, extraMode string) string {
```

Comment:
Why would it break . I added it so that it works for all envs. I don't want to have verbatim copies for each platform.